setup-vm.sh

```diff
@@ -31,6 +31,14 @@ then
   sed -E -i'' -e 's/(Defaults\s+requiretty)/#\1/' /etc/sudoers
 fi
 
+# Oracle
+if
+  [[ -f /etc/os-release ]] &&
+  GREP_OPTIONS="" \grep 'ID="ol"' /etc/os-release >/dev/null
+then
+  yum install libyaml-devel -y --enablerepo ol7_optional_latest
```

Comment:
this should be part of <https://github.com/rvm/rvm/blob/master/scripts/functions/requirements/centos>, steps here are related to quirks of setting up machines, not configuration for building ruby.